### PR TITLE
updates CI for use with GCP

### PR DESCRIPTION
**Pull request by wgahnagl**
Signed-off-by: Skyler Clark <skylerjaneclark@gmail.com>

#### What type of PR is this?

/kind ci 

#### What this PR does / why we need it:
updates the ansible to run on GCP without root
#### Does this PR introduce a user-facing change?

```release-note
None

```

### CODE REVIEW

File: contrib/test/ci/e2e-main.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
 - hosts: localhost, all
-  remote_user: root
+  become: yes
   environment:
     GOPATH: /usr/go
+    DIRPATH: ~/
```
Comment (sohankunkerkar) [inline contrib/test/ci/e2e-main.yml:6]: I don't think we need this anymore

File: contrib/test/ci/build/cri-o.yml [hunk 2]
```diff
@@ -8,12 +8,12 @@
 
 - name: check if cri-o.tar.gz exists
   stat:
-    path: /root/cri-o.tar.gz
+    path: "/home/deadbeef/cri-o.tar.gz"
```
Comment (sohankunkerkar) [inline contrib/test/ci/build/cri-o.yml:11]: How about `/home/foobar`?

Comment (wgahnagl) [reply contrib/test/ci/build/cri-o.yml:11]: I think it would be kinda weird to put it in a place that's not the homedir of the user

Comment (sohankunkerkar) [reply contrib/test/ci/build/cri-o.yml:11]: ah, my bad

**Comment by haircommander** [comment]
yolo

**Pull request merged**